src/Jackett.Common/Definitions/riperam.yml

```diff
@@ -922,6 +922,9 @@ search:
       filters:
         - name: querystring
           args: f
+    categorydesc|noappend:
+      selector: dt a:last-of-type
```

Comment:
what are you trying to achieve with this?
`dt a:last-of-type` will yield strings like `!!!   Полные завершенные сериалы   !!!` and `24` and `!!! Полные завершенные сезоны !!!`  and `Информационно-аналитические и общественно-политические передачи` and `Ultra HD (4K)` and  `60 минут` just to pick a few examples, none of which are relevant.

Reply:
categorydesc:noappend seems to work fine as an override for the default category.
I'll merge, but I am also going to follow up later with a category refresh from the site, as the indexer has not been  updated since Oct 2020.